internal/v1/storage/utils.go

```diff
@@ -1,6 +1,6 @@
 package storage
 
-var base = "https://cdn.denartcc.org/"
+var base = "https://cdn.zdvartcc.org/"
```

Comment:
This change is unnecessary, as the value is fed in through configuration. Ideally this should be refactored out since this API isn't just for ZDV any longer, but is replaced in main.go.